analyzers/src/SonarAnalyzer.CSharp/Helpers/CSharpDiagnosticAnalyzerContextHelper.cs

```diff
@@ -57,17 +57,9 @@ public static void ReportDiagnosticIfNonGenerated(this CompilationAnalysisContex
             }
         }
 
-        public static void ReportDiagnosticIfNonGenerated(this SymbolAnalysisContext context, Diagnostic diagnostic) =>
+        public static void ReportDiagnosticIfNonGenerated(this SonarSymbolAnalysisContext context, Diagnostic diagnostic) =>
```

Comment:
`ReportDiagnosticIfNonGenerated` at line 49 is not used anymore and can be removed as well.

Reply:
It's removed in #6545